### PR TITLE
Added support for Floats slices

### PR DESCRIPTION
**Pull request by mshtmfv**

### CODE REVIEW

File: redis/scan.go [hunk 1]
```diff
@@ -134,6 +150,8 @@ func convertAssignValue(d reflect.Value, s interface{}) (err error) {
 		err = convertAssignBulkString(d, s)
 	case int64:
 		err = convertAssignInt(d, s)
+	case float64:
```
Comment (garyburd) [inline redis/scan.go]: Explain why this is needed.  

Redigo does not use floats to represent Redis values.

File: redis/reply.go [hunk 2]
```diff
@@ -317,6 +317,22 @@ func Ints(reply interface{}, err error) ([]int, error) {
...
+
+// Floats is a helper that converts an array command reply to a []float64. If
+// err is not equal to nil, then Floats returns nil, err.
+func Floats(reply interface{}, err error) ([]float64, error) {
+	var floats []float64
+	values, err := Values(reply, err)
+	if err != nil {
+		return floats, err
+	}
+	if err := ScanSlice(values, &floats); err != nil {
```
Comment (garyburd) [inline redis/reply.go]: Convert the values directly instead of using ScanSlice.

Comment (mshtmfv) [reply redis/reply.go]: Why I can't use ScanSlice?

File: redis/scan.go (hunk 1 above)
Comment (mshtmfv) [reply redis/scan.go]: I need to store float lists in Redis so I need to get them back somehow to golang client.

File: redis/reply.go (hunk 2 above)
Comment (garyburd) [reply redis/reply.go]: Write it directly because the code is easier to understand and faster than ScanSlice.

Comment (mshtmfv) [reply redis/reply.go]: Ok, I would change code soon.

File: redis/scan.go (hunk 1 above)
Comment (garyburd) [reply redis/scan.go]: The Redis result types are error, simple string, bulk string, integer and array.  Redigo does not use float to represent any of these types.

**Pull request closed without merging**